app/utils/validateContent.js

```diff
@@ -0,0 +1,18 @@
+const AJV = require('ajv');
+const validator = new AJV();
+const v4MetaSchema = require('ajv/lib/refs/json-schema-draft-04.json');
+validator.addMetaSchema(v4MetaSchema);
+
+/**
+ * Validates the given content object with the given JSON schema
+ *
+ * @param      {object}  content  The content object
+ * @param      {object}  schema   The JSON schema
+ * @return     {Array}   [ isValid: boolean, errors: object ]
+ */
```

Comment:
Worth thinking about typescript or flow?

Reply:
probably not for this site, the JS is minimal

Reply:
Then why bother typing like this?

Reply:
because my doxygen comments generator plugin for sublime text generates this syntax, I think it's nice and informative for other devs wondering what the hell the function does.